server/src/main/java/org/jboss/as/server/logging/ServerLogger.java

```diff
@@ -115,6 +115,12 @@ public interface ServerLogger extends BasicLogger {
      */
     ServerLogger DEPRECATED_DEP_LOGGER = Logger.getMessageLogger(MethodHandles.lookup(), ServerLogger.class, "org.jboss.as.dependency.deprecated");
 
+    /**
+     * A logger with the category of {@code org.jboss.as.server.deployment.warning}.
+     * <strong>Usage:</strong> Use this in code related to deployment's warnings
+     */
+    ServerLogger DEPLOYMENT_WARN_LOGGER = Logger.getMessageLogger(MethodHandles.lookup(), ServerLogger.class, "org.jboss.as.server.deployment.warning");
+
```

Comment:
We should not create specific root loggers for specific levels. We create root loggers and then we configure the levels from outside, for example in a logger configuration file.

In this file there is already a logger for deployment, use it instead.